shuup/gdpr/models.py

```diff
@@ -5,14 +5,16 @@
 #
 # This source code is licensed under the OSL-3.0 license found in the
 # LICENSE file in the root directory of this source tree.
-from __future__ import unicode_literals
+from __future__ import unicode_literals, with_statement
```

Comment:
What are these `with_statement`?

Reply:
To enable `with reversion.create_revision():` in py2